### ISSUE TITLE
tiny readability fixes arising from a11y audit

### ISSUE THREAD

**Issue by StephDriver** [openlibhums/janeway#4214]
During screenreader testing (#4194) these small readability issues were found:

OLH Theme - [ Sample 1](https://ijwwe.uwp.co.uk/article/id/9711/)
Material Theme - [Sample 2] (https://www.glossa-journal.org/article/id/10082/)
Clean Theme - [Sample 3](https://politicalphilosophyjournal.org/article/id/15481/)

- [user content] OLH:  `Research Articles` as if a heading, but it is a single article on the page. Should be singular and a heading.
- [x] Material: Heading `Download` is ambiguous, 'Download Article' would be preferable. 
- [x] Clean: `Harvard-Style Citation` should be `Harvard-style Citation`. The only the first word of a hyphen joining should be capitalised.
- [user content] Clean: in the footer `published by Open Library of Humanities` according to new OLH style-guides there should be the definite article, so `published by the Open Library of Humanities`. 
- [fixed elsewhere ] Clean: in the footer `Login>` should be `Login`
- [ fixed elsewhere ] Clean homepage: the use of `[...]` at the end of a news article summary sets up an expectation of a link, but this is plain text.  Either make it a link, or `...` without the square brackets.
- [x] for all the above, check for other instances of similar and fix alongside.

**Comment by StephDriver** [comment]
from #4194 

- Sample 1, item 5. Research Articles is identified as Main and Landmark, [should this be singular not plural?] but not as any kind of heading.
- Sample 2, item 60. Download is consistent at H4 - but could be improved by 'download article' or similar, to avoid confusion. 
- Sample 3, item 47. "Harvard-Style Citation" should be "Harvard-style Citation"
- Sample 3, item 52. "published by Open Library of Humanities" should that be 'the'?
- Sample 3, item 53. "login greater than" for Login>
- Sample Other, item 7. in the footer where it is Published by Open Library of Humanities should that be Published by the Open Library of Humanities
